runtime/runtime/src/lib.rs

```diff
@@ -1426,6 +1436,8 @@ impl Runtime {
             tracing::trace!(target: "runtime", ?node_counter_before, ?node_counter_after);
 
             if let Some(outcome_with_id) = result? {
+                span.record("burnt_gas", outcome_with_id.outcome.gas_burnt);
```

Comment:
I believe this won't work as is the way you intend it to. The fields need to be "declared" in the original span definition before they can be recorded. See https://docs.rs/tracing/latest/tracing/field/struct.Empty.html and the front page docs at https://docs.rs/tracing/latest/tracing/index.html (grep for Empty).

Reply:
This has been added as part of https://github.com/near/nearcore/pull/10879, so you can simply drop this now.